docker/generic/start_proxy.py

```diff
@@ -498,12 +498,14 @@ def make_argparser():
         '''.format(creds_key=GOOGLE_CREDS_KEY))
 
     parser.add_argument(
-        '--dns_resolver_address',
+        '--dns_resolver_addresses',
         help='''
-        The IP address or the IP address followed with port number used to
-        resolve the domain name of backend. It should be in format of IP_ADDR
-        or IP_ADDR:PORT. For the IP_ADDR case, the default DNS port 52 will be
-        used.
+        The addresses of dns resolvers. Each address should be in format of
+        IP_ADDR or IP_ADDR:PORT and they are separated by '.';. For the IP_ADDR
```

Comment:
the example in the help text make me believe I can specify two ip addresses by
--dns_resolver_addresses=a.a.a.a  --dns_resolver_addresses=b.b.b.b

Is this supported,  if so,  it will be great. 

If no, please don't use this in the example.  

There is a typo in separated by '.'

Reply:
Done